oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -76,7 +76,7 @@ class ElasticIndexWriter implements FulltextIndexWriter<ElasticDocument> {
         // old index until the new one gets enabled) during incremental reindexing
         if (this.reindex) {
             try {
-                long seed = UUID.randomUUID().getMostSignificantBits();
+                long seed = indexDefinition.indexNameSeed == 0L ? UUID.randomUUID().getMostSignificantBits() : indexDefinition.indexNameSeed;
```

Comment:
Why not just use `new Random().nextLong()` or something else that generates only a long worth of randomness instead of a UUID?

Reply:
`new Random().nextLong()` might be okay but I don't see advantages compared to UUID. In case we want to change this we need to create a separate PR.